DEVGUIDE.md

```diff
@@ -116,10 +116,10 @@ Note: if you face this error [#2351](https://github.com/Microsoft/visualfsharp/i
 
 >  error VSSDK1077: Unable to locate the extensions directory. "ExternalSettingsManager::GetScopePaths failed to initialize PkgDefManager for C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.exe".
 
-delete those folders:
+Or hard crash on launch ("Unkown Error"), delete those folders:
 
-* %appdata%\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
-* %appdata%\..\Local\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
+* `%appdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
```

Comment:
this guy is in Local too.

I'm okay with either %Appdata% or %LocalAppdata%..  TBH I didn't even know about %LocalAppdata%